src/main/java/hudson/remoting/FlightRecorderInputStream.java

```diff
@@ -22,8 +22,8 @@ class FlightRecorderInputStream extends InputStream {
      * Size (in bytes) of the flight recorder ring buffer used for debugging remoting issues.
      * @since 2.41
      */
-    static final int BUFFER_SIZE =
-            Integer.getInteger("hudson.remoting.FlightRecorderInputStream.BUFFER_SIZE", 1024 * 1024);
+    static int BUFFER_SIZE =
```

Comment:
Left over when experimenting with changes to set the default to 0:

```suggestion
    static final int BUFFER_SIZE =
```